src/globals.ts

```diff
@@ -34,4 +34,14 @@ export const DEFAULTS = {
   PRIMARY_SALE: null,
 
   METADATA_PROGRAM_ID: 'metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s',
+
+  AIRTABLE_API_KEY: '',
```

Comment:
are you planning on adding these values here or in Vercel directly?  If in Vercel, LMK and I can help

Reply:
fixing globals.ts to follow this format like you requested earlier today in our message convo:

  PINATA_API_KEY:  process.env.VUE_APP_PINATA_API_KEY